apple/xcode/maplibre-rs.xcodeproj/project.pbxproj

```diff
@@ -481,6 +482,7 @@
 				);
 				"LIBRARY_SEARCH_PATHS[sdk=iphoneos*][arch=arm64]" = "$(PROJECT_DIR)/libs/aarch64-apple-ios/release";
 				"LIBRARY_SEARCH_PATHS[sdk=iphoneos*][arch=x86_64]" = "$(PROJECT_DIR)/libs/x86_64-apple-ios/release";
```

Comment:
Should we remove these search paths? What do you think? I think there is no x86_64 iphone.

Reply:
You're right, should I do this in the pull request?
We should probably support armv7(s) too but I don't know how it works or if it is compatible with arm64. https://docs.elementscompiler.com/Platforms/Cocoa/CpuArchitectures/#:~:text=iOS%20and%20iPadOS,-On%20iOS%20and&text=arm64%20is%20the%20current%2064,the%20A7%20and%20later%20chips.

Reply:
Yes I think you should remove the iphoneos x86 config in this PR.

Reply:
Yes armv7 armv7s and i386 support was dropped by Apple and Rust: https://blog.rust-lang.org/2020/01/03/reducing-support-for-32-bit-apple-targets.html